lib/react_client/component_factory.dart

```diff
@@ -367,34 +368,83 @@ class ReactDartFunctionComponentFactoryProxy extends ReactComponentFactoryProxy
 
   @override
   JsFunctionComponent get type => reactFunction;
+}
 
-  static String _getJsFunctionName(Function object) =>
```

Comment:
Use [whitespace-agnostic diff](https://github.com/cleandart/react-dart/pull/275/files?w=1) for this section